.github/workflows/main.yaml

```diff
@@ -31,4 +37,16 @@ jobs:
           TAG_NAME: edge
         run: |-
           REPO=${{ env.REPO }} VERSION=${{ env.TAG_NAME }} make plugin
+          docker plugin push "${{ env.REPO }}:${{ env.TAG_NAME }}-linux-amd64"
+          docker plugin push "${{ env.REPO }}:${{ env.TAG_NAME }}-linux-arm64"
           docker plugin push "${{ env.REPO }}:${{ env.TAG_NAME }}"
+
+          # docker currently does not support multi-arch plugins
+          #docker manifest create ${{ env.REPO }}:${{ env.TAG_NAME }} \
+          #  ${{ env.REPO }}:${{ env.TAG_NAME }}-linux-amd64 \
+          #  ${{ env.REPO }}:${{ env.TAG_NAME }}-linux-arm64
+          
+          #docker manifest annotate  ${{ env.REPO }}:${{ env.TAG_NAME }} ${{ env.REPO }}:${{ env.TAG_NAME }}-linux-amd64 --os linux --arch amd64
+          #docker manifest annotate  ${{ env.REPO }}:${{ env.TAG_NAME }} ${{ env.REPO }}:${{ env.TAG_NAME }}-linux-arm64 --os linux --arch arm64
+
+          #docker manifest push ${{ env.REPO }}:${{ env.TAG_NAME }}
```

Comment:
Unless this is something that'll work soon, I'd get rid of the commented out code here and in release.yaml